R/1_model_parameters.R

```diff
@@ -692,3 +692,8 @@ model_parameters.glm <- function(model,
   attr(out, "object_name") <- insight::safe_deparse_symbol(substitute(model))
   out
 }
+NULL
+#'	@title model_parameters.zoo
+#'	@description jr added Oct 17
+#'	@export
```

Comment:
```suggestion
#' @export
```

Reply:
Other directives are not needed here.

Reply:
Will do ...  late here so fix tomorrow;  need to check .Rd